### PR TITLE
Completed all P0 features

### PR DESCRIPTION
**Pull request by Pratham82**
- [x] Home Page
- [x] Quizzes by Category
- [x] Added topics page
- [x] Rules Page for while attempting Quiz
- [x] MCQ based Quiz Question Page
- [x] Quiz page for next question
- [x] Result Page with Answers & Score

UI Screenshots:
Home:
image

Topics page:
image

Quiz rules page:
image

Questions page:
image

Last question screen:
image

Results:
image

### CODE REVIEW

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good to me 
(i clicked user login it is not navigating to login page if it is not in P0 please ignore it)

**Review by masraniJay — APPROVED** [review]
Looks good.

File: src/components/quiz-next.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/Pratham82/min-ui@dev/src/css/minui.css"
+    />
+    <link rel="stylesheet" href="../css/index.css" />
+  </head>
+  <body>
+    <nav class="flex justify-around items-center navbar">
+      <div class="flex items-center branding">
+        <a href="../../../index.html">
```
Comment (shubhmehra) [inline src/components/quiz-next.html]: I think you can do '/index.html' instead of this '...//.../..'

**Review by shubhmehra — APPROVED** [review]
One small comment

**Comment by ashifkhn** [comment]
Looks Amazing.

File: src/components/quiz-next.html (hunk 1 above)
Comment (Pratham82) [reply src/components/quiz-next.html]: resolved

**Pull request merged**